gapic-generator-cloud/templates/cloud/gem/rakefile.erb

```diff
@@ -48,9 +48,10 @@ namespace :smoke_test do
   end
 end
 
+# rubocop:disable Metrics/BlockLength
```

Comment:
Rubocop complains when there's an unnecessary disable. Are we sure this disable is _always_ necessary? (Otherwise it's probably better to disable BlockLength for the Rakefile as a whole in the rubocop config.)

Reply:
Yes, but I expect this task to go away soon, and I wanted to keep the exclude near so that the exclude is removed when the task is.

Reply:
okay